sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -11,7 +15,7 @@
             v-model="legalName"
             data-cy="legalName"
             @keyup.enter="searchInstitutions"
-            :rules="[(v) => !!v || 'Legal name required']"
```

Comment:
i think we need rules

Reply:
we need rules at the common validation method, as only one input is required.